lib/modules/apostrophe-templates/views/outerLayoutBase.html

```diff
@@ -22,6 +22,8 @@
     {% block apostropheContextMenu %}
       {% if data.user %}
         <div class="apos-ui">
+          {# {{ apos.modules['apostrophe-ui'].tooltip() }} #}
+          {% include "apostrophe-ui:components/tooltip.html" %}
```

Comment:
Don't invalidate overrides of outerLauyoutBase unnecessarily - use `apos.templates.prepend` instead:

```
// in js land, not template land
apos.templates.prepend('contextMenu', function(req) {
  return self.partial('sometemplate.html', { some data... });
});
```

This is a clean way to push new template helpers to be run at certain points on the page without the need to keep adding stuff to outerLayoutBase. Not an absolute requirement for core module stuff like this feature, but it's clean and it provides bc with outerLayoutBase overrides new enough to include the apos.templates.prepended calls.

Reply:
calling this from `apostrophe-ui`'s `construct` method now